cfgov/v1/jinja2/v1/_includes/organisms/image-and-text-50-50.html

```diff
@@ -1,20 +1,63 @@
-
 {# ==========================================================================
 
-   image_and_text_50_50.render()
+   image_and_text-50-50.render()
```

Comment:
Fixed to `image-and-text-50-50` which is the convention I saw other modules using in 3db1e45